tests/BenchmarkDotNet.Tests/Exporters/XmlSerializerTests.cs

```diff
@@ -0,0 +1,245 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using BenchmarkDotNet.Exporters.Xml;
+using Xunit;
+
+namespace BenchmarkDotNet.Tests.Exporters
+{
+    public class XmlSerializerTests
+    {
+        [Fact]
+        public void Serialize()
+        {
+            string expected =
```

Comment:
nit: you can make `expected` output a `const` if it's a string. Then you are sure that your code is not changing the value.